### ISSUE TITLE
Fix Australia GPU nodes

### ISSUE THREAD

**Issue by AquiGorka** [Lilypad-Tech/lilypad#61]

**Comment by AquiGorka** [comment]
How is work coming along on this task?

**Comment by Zorlin** [comment]
This is done, at least for now it appears to be all correct and working.

I'm unclear on what the major blocker was last time I attempted this - last time I tried it *it would not work no matter what* despite successfully using Docker to run the exact same jobs Lilypad and Bacalhau were running - and even getting output!

Inferno has been confirmed to be running jobs successfully (the A4000 node) albeit sometimes a bit slow? (as bad as a 7 minute run in one case)

The steps I took were, for history's sake:
* Checked all permissions and that everything was owned by the bacalhau user and group
* Checked that every node specified how much RAM it has to offer in both Bacalhau *and* the RP (important for it to get any jobs)
* In Monstar's case updated the GPU drivers and fixed a few package issues; as nvidia-smi was reporting no GPU, fixed that
* Updated everyone to latest Lilypad and Bacalhau v1.2.3 (or rather, verified we were on Bac 1.2.3 as those nodes were already using that version)

Monstar is also running jobs but I haven't been able to directly observe one. I suspect that the solver picks higher RAM nodes first, such as the 4090 (24GB) and A4000 (16GB) nodes, so we won't observe it running jobs all that often except when the network is busy.
